ui/src/app.js

```diff
@@ -650,14 +656,135 @@ function setWifiList (recs, requestedPage) {
+    } else {
+      lineStartIndex = 0
+      lineEndIndex = calculateLinesLastWordIndex(textArray, lineEndIndex)
+    }
+    linesArray.push(createLine(textArray, lineStartIndex, lineEndIndex))
+  }
+  return linesArray
+}
+
+function splitToPages (linesArray) {
+  var currentPageLineIndex = 0
+  var pageArray = []
+
+  pageArray.push(linesArray.slice(0, MAX_LINES_IN_PAGE).join(' '))
+  var pageText = linesArray[MAX_LINES_IN_PAGE - 1]
+  
+  for (var i = MAX_LINES_IN_PAGE; i < linesArray.length; ++i) {
+    if (currentPageLineIndex === MAX_LINES_IN_PAGE - 1) {
+      currentPageLineIndex = 0
+      pageArray.push(pageText)
+      pageText = linesArray[i - 1]
+    }
+    pageText = pageText.concat(' ' + linesArray[i])
+    currentPageLineIndex++
+  }
+  if (currentPageLineIndex !== 0) {
+    pageArray.push(pageText)
+  }
+
+  return pageArray
+}
+
+function renderTermsText (text) {
+  text = removePTagFromText(text)
+  var textArray = text.split(' ')
+  var linesArray = splitToLines(textArray)
+  termsPages = splitToPages(linesArray)
+  startPage()
+}
+
+function createLine (textArray, startWordIndex, endWordIndex) {
+  return '<h5>' + textArray.slice(startWordIndex, endWordIndex + 1).join(' ') + '</h5>'
+}
+
+function calculateLinesLastWordIndex (words, oldEndIndex) {
+  var newEndIndex = oldEndIndex
+  var line = words[oldEndIndex]
+  for (var i = oldEndIndex + 1; i < words.length; ++i) {
+    var sentence = line.concat(' ' + words[i])
+    if (sentence.length <= MAX_CHARACTERS_IN_LINE) {
+      line = sentence
+      newEndIndex = i
+    } else {
+      return newEndIndex
+    }
+  }
+  return Math.max(oldEndIndex, newEndIndex)
+}
+
+// click page up button
+function scrollUp () {
+  currentPageIndex = Math.max(0, currentPageIndex -= 1)
+  var termsText = termsPages[currentPageIndex]
+  updateButtonStyles(currentPageIndex)
+  var $screen = $('.js-terms-screen')
+  $screen.find('.js-terms-text').html(termsText)
+}
+
+// start page
+function startPage () {
+  var $screen = $('.js-terms-screen')
+  var currentPageText = termsPages[currentPageIndex]
+  updateButtonStyles(currentPageIndex, termsPages.length - 1)
+  $screen.find('.js-terms-text').html(currentPageText)
+}
+
+// click page up button
+function scrollDown () {
+  currentPageIndex = Math.min(termsPages.length - 1, currentPageIndex += 1)
+  var termsText = termsPages[currentPageIndex]
+  updateButtonStyles(currentPageIndex, termsPages.length - 1)
+  var $screen = $('.js-terms-screen')
+  $screen.find('.js-terms-text').html(termsText)
+}
+
+function updateButtonStyles (currentPageIndex, border) {
+  var buttonDown = document.getElementById('scroll-down')
+  var buttonUp = document.getElementById('scroll-up')
+  if (currentPageIndex === 0) {
+    makeButtonDisabled(buttonUp)
+  } else {
+    makeButtonEnabled(buttonUp)
+  }
+  if (currentPageIndex === border) {
+    makeButtonDisabled(buttonDown)
+  } else {
+    makeButtonEnabled(buttonDown)
+  }
+}
```

Comment:
Buttons should not appear at all (not even disabled) if text completely fits into the screen.